CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Bug fixes
 
+* [#6821](https://github.com/rubocop-hq/rubocop/pull/6821): Fix false negative for Rails/LinkToBlank when _blank is a symbol. ([@Intrepidd][])
```

Comment:
This is a nitpick. It seems that `_` intents italic start if `_blank` is not enclosed in backticks.

```diff
-* [#6821](https://github.com/rubocop-hq/rubocop/pull/6821): Fix false negative for Rails/LinkToBlank when _blank is a symbol. ([@Intrepidd][])
+* [#6821](https://github.com/rubocop-hq/rubocop/pull/6821): Fix false negative for Rails/LinkToBlank when `_blank` is a symbol. ([@Intrepidd][])
```

Reply:
Thanks, fixed